contrib/edu-examples/tensorflow/README.md

```diff
@@ -0,0 +1,25 @@
+This folder includes some configuration examples for submmitting jobs to OPEN PAI platform. 
+
+Here we provides 4 examples, the examples use VGG16 model on the CIFAR-10 task, trained with 
+- Single CPU: cifar10_vgg16_tf_cpu.yaml
+- Single GPU (Tesla K80): cifar10_vgg16_tf_gpu.yaml
+- 4 GPU (with tensorflow native distributed training): cifar10_vgg16_tf_gpu.yaml
+- 4 GPU (with horovod): cifar10_vgg16_tf_gpu.yaml 
+
+The performance of the tasks with different hyper-parameters (batch-size) are shown in the table below:
+
+With batch-size=32:
+|  mode | accuracy | run time | job level metrics |
+| ------------- | ------------- | ------------- | ------------- |
+| single CPU  | 97.29% | 5h 3m | [link](https://github.com/microsoft/pai/bolb/pai-for-edu/contrib/edu-examples/tensorflow/metrics/1cpu_32.png) |
```

Comment:
Please use a relative path in the link, like `./metrics/...png`

Reply:
fixed